Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt

```diff
@@ -309,27 +307,23 @@ private fun UserInputSelector(
         }
         Spacer(modifier = Modifier.weight(1f))
 
-        val backgroundColor = ButtonConstants.defaultButtonBackgroundColor(
-            enabled = sendMessageEnabled,
-            disabledColor = MaterialTheme.colors.surface
-        )
         val disabledContentColor =
             AmbientEmphasisLevels.current.disabled.applyEmphasis(MaterialTheme.colors.onSurface)
-        val contentColor = ButtonConstants.defaultButtonContentColor(
-            enabled = sendMessageEnabled,
-            defaultColor = contentColorFor(backgroundColor),
-            disabledColor = disabledContentColor
+
+        val buttonColors = ButtonConstants.defaultButtonColors(
+            disabledBackgroundColor = MaterialTheme.colors.surface,
+            disabledContentColor = disabledContentColor
         )
+
         // Send button
         Button(
             modifier = Modifier
                 .padding(horizontal = 16.dp)
                 .preferredHeight(36.dp),
-            elevation = 0.dp,
+            elevation = ButtonConstants.defaultElevation(defaultElevation = 0.dp),
```

Comment:
Default is `0.dp` already

```suggestion
            elevation = ButtonConstants.defaultElevation(),
```

Reply:
The default is the default which is 0 so apparently we can remove the whole thing. Strange that I needed to add it in the first place